cosmo_tester/test_suites/snapshots/__init__.py

```diff
@@ -55,6 +56,33 @@
     TENANT_DEPLOYMENTS_PATH + '/{name}'
 )
 
+MANAGER_VERSIONS_LIST = (
+    '3.4.2',
+    '4.0',
+    '4.0.1',
+    '4.1',
+    'master'
+)
+# Only test 3.4.2 and 4.0 with a single tenant on premium.
+# Everything else should work with multi tenant
+MANAGER_SINGLE_TENANT_SLICE = 2
```

Comment:
this slicing looks pretty fragile to me, will we need to later keep this in sync if the tuple/list changes? Probably not, since future versions SHOULD work with multi tenant of course... but eh, slicing seems just like a weird way to split the list in two.
Why not just have two separate versions lists, `SINGLE_TENANT_VERSIONS` and `MULTI_TENANT_VERSIONS` or something like that. (and use `+` for community-singletenant)

Bonus points if the list doesn't look too general - we probably don't want someone in the future this here is the official list of manager versions, and importing it into some other (non-snapshots) test...